src/index.html

```diff
@@ -7,7 +7,252 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <article class="header__logoliner">
+        <section class="header__logoliner-1">
+          +1 234 555 55 55
+        </section>
+        <section class="header__logoliner-2">
+          <img
+            class="header__logo"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </section>
+        <section class="header__logoliner-3">
+          <a href="#">CALL TO ORDER</a>
+        </section>
+      </article>
+      <input
+        id="menu"
+        class="menu"
+        type="checkbox"
+        hidden
+      >
+      <article class="header__logoliner-mobile">
+        <img
+          class="header__logoliner-1"
+          src="images/phone.svg"
+          alt="phone"
+        >
+        <img
+          class="header__logoliner-2"
+          src="images/logo.png"
+          alt="logo"
+        >
+        <label for="menu">
+          <img
+            class="header__logoliner-3"
+            src="images/menu.svg"
+            alt="menu"
+          >
+        </label>
+      </article>
+      <ul>
+        <li>
+          <a href="#">HOME</a>
+        </li>
+        <li>
+          <a href="#">ABOUT US</a>
+        </li>
+        <li>
+          <a href="#">GALLERY</a>
+        </li>
+        <li>
+          <a href="#">CONSULTATION</a>
+        </li>
+        <li>
+          <a href="#">TESTIMONIALS</a>
+        </li>
+        <li>
+          <a href="#">CONTACTS</a>
+        </li>
+        <li><a href="#">+ 1 234 555-55-55</a></li>
+        <li class="header__logoliner-3">
+          <a href="#">CALL TO ORDER</a>
+        </li>
+      </ul>
+      <article class="header__title">
+        <h1 class="header__text">Miami Condo Kings</h1>
+      </article>
+      <nav class="header__navigation">
+        <a class="header__navigation-link" href="#">HOME</a>
+        <a class="header__navigation-link" href="#">ABOUT US</a>
+        <a class="header__navigation-link" href="#">GALLERY</a>
+        <a class="header__navigation-link" href="#">CONSULTATION</a>
+        <a class="header__navigation-link" href="#">TESTIMONIALS</a>
```

Comment:
тут і далі в html слід тег і текст розміщувати на окремих рядках
```suggestion
        <a class="header__navigation-link" href="#">
          TESTIMONIALS
        </a>
```